Libraries/LibWeb/Bindings/XMLHttpRequestPrototype.cpp

```diff
@@ -40,7 +40,14 @@ XMLHttpRequestPrototype::XMLHttpRequestPrototype()
 {
     put_native_function("open", open, 2);
     put_native_function("send", send, 0);
+    put_native_property("readyState", ready_state_getter, nullptr);
     put_native_property("responseText", response_text_getter, nullptr);
+
+    put("UNSENT", JS::Value((i32)XMLHttpRequest::ReadyState::Unsent));
+    put("OPENED", JS::Value((i32)XMLHttpRequest::ReadyState::Opened));
+    put("HEADERS_RECEIVED", JS::Value((i32)XMLHttpRequest::ReadyState::HeadersReceived));
+    put("LOADING", JS::Value((i32)XMLHttpRequest::ReadyState::Loading));
+    put("DONE", JS::Value((i32)XMLHttpRequest::ReadyState::Done));
```

Comment:
Apparently these have to exist both on the constructor and prototype, presumably to make this possible:

```js
const xhr = new XMLHttpRequest();
if (xhr.readyState === xhr.DONE) ...
if (xhr.readyState === XMLHttpRequest.DONE) ...
```